### ISSUE TITLE
[macOS] Can't Build minikube (0.22.3)

### ISSUE THREAD

**Issue by chriskaukis** [NixOS/nix#1676]
TL;DR: `nix-env -i minikube` fails to build on macOS (10.13.1).

There doesn't seem to be any specific notes or requirements. I'm only starting to get into using Kubernetes, Docker, and what ever else is involved, so I'm not the most intelligent or familiar with all that is involved still. Perhaps if this is a missing package on my part Nix can provide or mention in description or notes what options there are? Just a thought. As it seems to fail on a *drv file or some sorts I'm assuming it's missing a driver I need to install.

I can help and deep dive as much as possible, but only have so much time. If you can provide me with what or where to look I'm more than happy to contribute. For now, I have attached the script file of the command and output to console ¯\_(ツ)_/¯ if it helps. I'm trying to anyway. Please let me know what I can do to help contribute to make this work or easier for me or others in the future.

[script.txt](https://github.com/NixOS/nix/files/1467224/script.txt)

(Wasn't sure how to file a bug/issue. Some projects have specific wiki/docs setup with requirements. I didn't see one here.)

**Comment by copumpkin** [comment]
I think if you add `darwin.apple_sdk.frameworks.vmnet` to its `buildInputs`, that should fix it.

**Comment by copumpkin** [comment]
Btw this belongs in the nixpkgs repo, not the nix repo, which only handles the Nix package manager program rather than the packages it manages.

**Comment by chriskaukis** [comment]
Sorry. Off the top I'm not familiar enough yet to understand how to set buildInputs. I don't see it documented as an argument. It looks like I would have to download the package and edit the install configuration and install from that.
